src/services/study/network.js

```diff
@@ -132,6 +132,42 @@ export function getSubstationSingleLineDiagram(
     );
 }
 
+export function fetchNetworkElementsIds(
```

Comment:
Why not using the existing `fetchEquipmentsIds` in src/services/study/network-map.js

Reply:
done , but i discovred a bug in 'fetchEquipmentsIds in src/services/study/network-map.js so i did some changes to it

Reply:
Keep second implementation and plan a refactor of the previous function to avoid impact on other functionalities.